File: CSharp version/it/unical/mat/embasp/languages/asp/SymbolicConstant.cs
namespace EmbASP.it.unical.mat.embasp.languages.asp
{
    class SymbolicConstant
    {
        private string value;
        public SymbolicConstant() { }

        public SymbolicConstant(string value)
        {
            this.Value = value;
        }

        public string Value { get => value; set => this.value = value; }

        public override string ToString()
        {
            return Value;
        }
    }
}

File: Java version/app/src/main/java/it/unical/mat/embasp/languages/asp/SymbolicConstant.java
package it.unical.mat.embasp.languages.asp;

public class SymbolicConstant {
	
	private String value;
	
	public SymbolicConstant() {
		value = "";
	}
	
	public SymbolicConstant(String value) {
		this.value = value;
	}
	
	public String getValue() {
		return value;
	}
	
	public void setValue(String value) {
		this.value = value;
	}
	
	@Override
	public String toString() {
		return value;
	}

}
